### PR TITLE
New hub fields for playercount & real round time

### PR DESCRIPTION
**Pull request by craftxbox**
### Purpose
Reports three new fields to the current hub api for future use, 
RoundTime: Mirror of GameManager.Instance.RoundTimeInMinutes, Gives an accurate display of how long a round has gone on for (that doesn't include janky math on the existing IngameTime variable). This field is OPTIONAL and may not appear on the API if it's not being reported.
PlayerCountMax: Should be pretty self-explanatory. If it's not reported, this value is 0 through the api.
Passworded: ditto. If a connection password is set, this is true. Else false (also if not reported)

### Changelog:
Not userfacing, No CL.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Core/Database/ServerData.ServerStatus.cs [hunk 1]
```diff
@@ -161,6 +161,10 @@ IEnumerator SendServerStatus()
 				status.CurrentMap = SubSceneManager.ServerChosenMainStation;
 			}
 
+			status.Passworded = !string.IsNullOrEmpty(config.ConnectionPassword);
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Core/Database/ServerData.ServerStatus.cs]: == false

**Comment by Bod9001** [comment]
does this need any other PRs to be Merged first?

File: UnityProject/Assets/Scripts/Core/Database/ServerData.ServerStatus.cs (hunk 1 above)
Comment (craftxbox) [reply UnityProject/Assets/Scripts/Core/Database/ServerData.ServerStatus.cs]: I can do that but that doesn't seem very easy to understand in a variable declaration

**Comment by craftxbox** [comment]
> does this need any other PRs to be Merged first?

Nope, This makes fields available for stationhub to read should it want to.

File: UnityProject/Assets/Scripts/Core/Database/ServerData.ServerStatus.cs (hunk 1 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/Core/Database/ServerData.ServerStatus.cs]: thouggh it is coding practice to do == false, 
I think it's fine with stuff like !NoPassword, 
where it is a direct inverse of a bool that is already there

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**